packages/workspace/src/core/affected-project-graph/locators/implicit-json-changes.ts

```diff
@@ -54,16 +56,18 @@ function getTouchedProjectsByJsonFile(
   return projects;
 }
 
-function getTouchedProjects(path: string[], implicitDependencyConfig: any) {
-  let curr = implicitDependencyConfig;
-  let found = true;
-  for (const key of path) {
-    if (curr[key]) {
-      curr = curr[key];
-    } else {
-      found = false;
-      break;
+function getTouchedProjects(
+  path: string[],
+  implicitDependencyConfig: any
+): string[] {
+  const flatConfig = flatten(implicitDependencyConfig, { safe: true });
```

Comment:
Can we do this without flattening the config?

Can we expand the glob within `normalizeImplicitDependencies`?